core/includes/install.core.inc

```diff
@@ -1589,6 +1589,7 @@ function install_import_translations(&$install_state) {
     $language = (object) array(
       'langcode' => $langcode,
       'name' => $langcode,
+      'native' => $langcode,
```

Comment:
While at it, would you please change the `The user will be able to edit afterwards.` bit in the inline comment above to something like `If required, this can be edited post-installation.` instead?